### ISSUE TITLE
Mac release for v0.11.2 not working

### ISSUE THREAD

**Issue by tcl16** [bastimeyer/livestreamer-twitch-gui#200]
When attempting to launch the extracted .app from both the 32 bit and 64 bit mac releases of v0.11.2, I get the message 
> The application “Livestreamer Twitch GUI 0.11.2.app” can’t be opened.

The 64 bit release of v0.11.1 works perfectly fine on the same system. The permissions on Livestreamer\ Twitch\ GUI\ 0.11.2.app/Contents/MacOS/nwjs seem to be incorrect because I believe it should be executable as it is in v0.11.1. 
```
[~]$ ls -l Downloads/Livestreamer\ Twitch\ GUI\ 0.11.1.app/Contents/MacOS/nwjs
-rwx------@ 1 t  staff  8552 Dec  5 05:45 Downloads/Livestreamer Twitch GUI 0.11.1.app/Contents/MacOS/nwjs
[~]$ ls -l Downloads/Livestreamer\ Twitch\ GUI\ 0.11.2.app/Contents/MacOS/nwjs
-rw-r--r--@ 1 t  staff  8552 Jan 23 10:53 Downloads/Livestreamer Twitch GUI 0.11.2.app/Contents/MacOS/nwjs
```

However this permission is not the only problem because adding the executable flag does not fix the issue.

**Comment by bastimeyer** [comment]
Oh boy... :sob: 
The build system didn't change, so I'm not sure why the file permissions are not working... Let me see...
Thanks for reporting, though

**Comment by bastimeyer** [comment]
Ok, I have found the issue...
I just recently copied my repo with all the cached NWjs files to a new partition on my system and somehow the file permissions were not set correctly. I did not expect this... :disappointed: 

Easy fix: I don't need to change anything, since redownloading NWjs will fix this issue. I will recompile the files from the `v0.11.2` tag and then reupload the files. Give me a couple of mins, so I can check that evrything is working again....
